app/sw/registerSW.ts

```diff
@@ -0,0 +1,34 @@
+// To active execute the following code in the browser console and refresh:
+// localStorage.setItem("installServiceWorker", true)
+
+if ("serviceWorker" in navigator) {
```

Comment:
Any reason we wouldn't just keep this in app/index as before? Avoid the additional entry point and plumbing.

Reply:
My understanding was:

1. The HTML is served through getoutline.com.
2. The service worker must be registered through JS (inline or registerSW.js) from the same domain.
3. And the actual service worker must be served through the same domain.

But I seem to be wrong. Currently, you’re serving assets from the CDN and it’s working. 🤦‍♂️ I’ll give it another try.